tools/infer/utility.py

```diff
@@ -293,6 +299,28 @@ def create_predictor(args, mode, logger):
             config.enable_custom_device("mlu")
         elif args.use_xpu:
             config.enable_xpu(10 * 1024 * 1024)
+        elif args.use_gcu:  # for Enflame GCU(General Compute Unit)
+            import paddle_custom_device.gcu.passes as gcu_passes
```

Comment:
Should we first check whether the `paddle_custom_device` package is present? If it’s not, remind the user to install it first?

Reply:
Thank you for your suggestion.